service-commands/scripts/setup-instance.sh

```diff
@@ -138,25 +99,26 @@ case "$service" in
 			"[[ ! -d ~/audius-protocol ]]" \
 			"&& set -x" \
 			"&& git clone https://github.com/AudiusProject/audius-protocol.git" \
-			"&& PROTOCOL_DIR=/home/ubuntu/audius-protocol bash ~/audius-protocol/service-commands/scripts/set-git-refs.sh $audius_protocol_git_ref $audius_client_git_ref" \
-			"&& yes | bash audius-protocol/service-commands/scripts/provision-dev-env.sh" || true
-        reboot_instance $provider $name
+			"&& PROTOCOL_DIR=audius-protocol bash ~/audius-protocol/service-commands/scripts/set-git-refs.sh $audius_protocol_git_ref $audius_client_git_ref" \
+			"&& yes | bash audius-protocol/service-commands/scripts/provision-dev-env.sh"
+		reboot_instance $provider $name
 
 		wait_for_instance $provider $user $name
 
 		# perform once more for audius-client
 		execute_with_ssh $provider $user $name \
-			"PROTOCOL_DIR=/home/ubuntu/audius-protocol bash ~/audius-protocol/service-commands/scripts/set-git-refs.sh $audius_protocol_git_ref $audius_client_git_ref"
+			"bash ~/audius-protocol/service-commands/scripts/set-git-refs.sh $audius_protocol_git_ref $audius_client_git_ref"
 
-		execute_with_ssh $provider $user $name "nohup npm run start:dev:cloud > ~/audius-client.out 2>&1 &"
-		execute_with_ssh $provider $user $name "source ~/.nvm/nvm.sh; source ~/.profile; A up || (A down && A up)"
-
-		copy_file_to_remote $provider $user $name "~/.gitconfig" "~/.gitconfig"
+		# start up frontend and backend
+		execute_with_ssh $provider $user $name "set -x; nohup npm run start:dev:cloud > ~/audius-client.out 2>&1 &"
```

Comment:
Have you tested this? I tried this before but I don't think this will work since ssh doesn't open up an interactive shell and therefore doesn't load user dotfiles - you'll get `npm not found`.

Reply:
Yeah, not sure how, but now I can't not run into this error, but even my AM run went smoothly. But +1 on removing it.

I looked into execute_with_ssh further, but couldn't get nvm loaded using `gssh` nor `ssh` even though there were no problems with this yesterday/last night. :/